src/main/java/org/jenkinsci/test/acceptance/plugins/dashboard_view/LatestBuildsPortlet.java

```diff
@@ -0,0 +1,117 @@
+package org.jenkinsci.test.acceptance.plugins.dashboard_view;
+
+import org.jenkinsci.test.acceptance.po.Control;
+import org.jenkinsci.test.acceptance.po.Describable;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+
+/**
+ * The basic latest builds portlet shipped with the dashboard view plugin.
+ *
+ * @author Rene Zarwel
+ */
+@Describable("Latest builds")
+public class LatestBuildsPortlet extends AbstractDashboardViewPortlet {
+
+    private Control numberOfBuilds = control("numBuilds");
+
+    /**
+     * Default name of this Portlet.
+     */
+    public static final String PORTLET_NAME = "Latest builds";
+    /**
+     * Default number of latest builds to show.
+     */
+    public static final int NUMBER_OF_BUILDS = 10;
+
+
+    /**
+     * Constructs a new latest builds portlet.
+     *
+     * @param parent Parent dashboard view this portlet is scoped to.
+     * @param path   Absolute path to the area.
+     */
+    public LatestBuildsPortlet(DashboardView parent, String path) {
+        super(parent, path);
+    }
+
+    /**
+     * Gets the latest builds table as {@link WebElement}.
+     *
+     * @return latest builds table
+     */
+    public WebElement getTable() {
+        WebElement portlet = find(By.xpath("//div[contains(.,'" + PORTLET_NAME + "')]/following::table[1]"));
+
+        return portlet.findElement(By.id("statistics"));
+    }
+
+    private WebElement getRow(int row) {
+        return getTable().findElement(By.xpath(".//tbody/tr[" + row + "]"));
+    }
+
+    /**
+     * Sets the number of builds of the portlet.
+     *
+     * @param number the number of builds of the portlet
+     */
+    public void setNumberOfBuilds(int number) {
```

Comment:
One can manually input bad values in the view as well, so i wouldn't restrict that. Maybe a test will especially check the behaviour on bad values in the future